go.mod

```diff
@@ -3,35 +3,30 @@ module github.com/cosmos/gaia/v9
 go 1.20
 
 require (
-	github.com/cosmos/cosmos-sdk v0.45.15-ics
+	cosmossdk.io/math v1.0.0
+	github.com/cosmos/cosmos-sdk v0.45.15
 	github.com/cosmos/go-bip39 v1.0.0
-	github.com/cosmos/ibc-go/v4 v4.2.0
-	github.com/cosmos/interchain-security v1.1.0
+	github.com/cosmos/ibc-go/v4 v4.3.0
+	github.com/cosmos/interchain-security v1.2.0
```

Comment:
Please keep the PR contained to one issue. There is no need to bump the version of ICS in the same PR where we bump ibc-go.

Reply:
https://github.com/cosmos/interchain-security/blob/6a856d183cd6fc6f24e856e0080989ab53752102/go.mod#L31

there is.  At least, while I do understand that gaia's version will dominate, I've seen a lot of kaboomies.

To prevent them I like to ensure that dependencies use the exact same versions of ecosystem libraries

Reply:
I'll revert, but wish to note that I think it is safer to group this stuff

Reply:
@faddat Thank you for sharing your concern. If you think that bumping ibc-go to v4.3.0 requires a bump of ICS as well, please open an issue describing the problem in details. We can discuss your concerns on that issue and decide on how to best address them. 

From the description and title of this PR, the goal is to 
> Removes the now-unnecessary dragonberry replace and updates to ibc-go v4.3.0

Adding multiple changes to the same PR without properly documenting it, will lead to difficulties in maintaining Gaia in the long run. For example, `github.com/cosmos/interchain-security v1.2.0` contains multiple changes that need to be properly tested before adding them to Gaia. Moreover, [v1.2.0 still uses ibc-go v4.2.0](https://github.com/cosmos/interchain-security/blob/1368b95c0b4e9a1e7e18ace4bbfec1d147c53c9b/go.mod#L8).